scss/civicrm/common/_modals.scss

```diff
@@ -71,7 +68,12 @@
       }
 
       .ui-icon[class*=" fa-"] {
-        margin-left: -8px !important;
+        font-size: $font-size-h2;
+        height: $font-size-h1;
```

Comment:
I am not sure using a variable stating "font-size" for "height" is a good idea. I think we should have a separate variable, see above.